### ISSUE TITLE
It "Just works" in Firefox. Please port it there also!

### ISSUE THREAD

**Issue by andreicristianpetcu** [checkmate-bitch/QWiki#5]
You can port it either with [CSF](https://addons.mozilla.org/en-US/firefox/addon/chrome-store-foxified/) or by loading the manifest.json in Firefox's `about:debugging` page.

Thank you!

**Comment by checkmate-bitch** [comment]
Thank You for the tip. I was a little busy. I'll do that as soon as I can.

**Comment by andreicristianpetcu** [comment]
I can try to publish it to AMO and if it gets approved, I can transfer it to you. If there are any changes needed, I will send a pull request. All you need to do is signup to AMO so I will have an account to transfer the WebExtension to.

**Comment by checkmate-bitch** [comment]
I'm just working on this repo. That's about to finish. Then I'll check out AMO and let you know once I sign up.

**Comment by checkmate-bitch** [comment]
Yeah, what do I have to do? Btw, finally was able to get to a satisfactory level with this extension,

**Comment by andreicristianpetcu** [comment]
1) Sign up for a Firefox account
2) Zip the extension and upload it [here](https://addons.mozilla.org/developers/addon/submit/distribution) and select "On this site".
3) Wait for reviewer feedback. You need to respect the items in the [Listed column](https://developer.mozilla.org/Add-ons/AMO/Policy/Reviews)
